libs/shared/ui/src/lib/components/table/table.stories.tsx

```diff
@@ -3,33 +3,51 @@ import { environmentFactoryMock } from '@console/domains/environment'
 import { Table, TableProps } from './table'
 import { TableRow } from './table-row/table-row'
 import { useState } from 'react'
+import Button from '../buttons/button/button'
+import { EnvironmentEntity } from '../../../../../interfaces/src'
```

Comment:
weird import should be @console/interface ? or you add this to avoid circular dependencies ?